### ISSUE TITLE
Unwanted migrations after upgrading to django 1.9

### ISSUE THREAD

**Issue by itbabu** [divio/django-cms#5011]
Hello! This will take a while :)

Django-cms  triggers unwanted migrations after upgrading to Django 1.9.
 
Here a django cms project for reproduce this behaviour: https://github.com/itbabu/test-django-cms

The important parts
--------------------------
1.  in settings.py:

    ````Python
    LANGUAGE_CODE = 'it'
    USE_I18N = True
    LANGUAGES = (
        ('it', gettext('it')),
    )
    ````
2. The apphook 
3. The ``demo_app``` with only english strings inside the model and the italian locale files.

Steps
--------
* Install this project (virtualenv with Python 3.5, ```pip install -r requirements.txt```)
* use ```project.db``` included in the repository or create a new database.
* run ```makemigrations demo_app``` with django 1.8.9 (you get "No changes detected")
* pip install Django==1.9.2
*  run ```makemigrations demo_app``` 
```
Migrations for 'demo_app':
  0002_auto_20160216_2129.py:
    - Change Meta options on season
    - Alter field description on season
    - Alter field order on season
    - Alter field title on season
```
The unwanted migration file created (look at ```verbose_name``` and ```verbose_name_plural```)
```Python
# -*- coding: utf-8 -*-
# Generated by Django 1.9.2 on 2016-02-16 20:29
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('demo_app', '0001_initial'),
    ]

    operations = [
        migrations.AlterModelOptions(
            name='season',
            options={'ordering': ['order'], 'verbose_name': 'stagione', 'verbose_name_plural': 'stagioni'},
        ),
        migrations.AlterField(
            model_name='season',
            name='description',
            field=models.CharField(max_length=300, verbose_name='descrizione'),
        ),
        migrations.AlterField(
            model_name='season',
            name='order',
            field=models.PositiveIntegerField(default=0, verbose_name='ordine'),
        ),
        migrations.AlterField(
            model_name='season',
            name='title',
            field=models.CharField(max_length=100, verbose_name='titolo'),
        ),
    ]

```

I did some debug:

I did git bisect for Django: django-cms triggers the migrations from [this Django commit](https://github.com/django/django/commit/fe3fc5210f0bb334a679ed420152af1c862c0239). It's related to urls.

I think that the django-cms lines responsible are [these](https://github.com/divio/django-cms/blob/develop/cms/appresolver.py#L211-L228) related to app hooks: you can comment them and run ```makemigrations``` and no migration is created.

**Comment by yakky** [comment]
Thanks @itbabu , apparently migrations takes translations into account if models are loaded by cms app_resolver

**Comment by yakky** [comment]
@itbabu could you test linked PR? It should fix the issue

**Comment by yakky** [comment]
Thanks @itbabu. PR has been merged and it will be released as part of 3.2.2
